docs/book/customers/map.rst.inc

```diff
@@ -1,8 +1,4 @@
-.. raw:: html
-
-    <ul class="simple">
-    <li><p><a class="reference internal" href="customers/customer_and_shopuser.html"><span class="doc">Customer and ShopUser</span></a></p></li>
-    <li><p><a class="reference internal plus-feature" href="customers/customer_pools.html"><span class="doc">Customer Pools</span></a></p></li>
-    <li><p><a class="reference internal" href="customers/adminuser.html"><span class="doc">AdminUser</span></a></p></li>
-    <li><p><a class="reference internal" href="customers/addresses/index.html"><span class="doc">Addresses</span></a></p></li>
-    </ul>
\ No newline at end of file
+* :doc:`/book/customers/customer_and_shopuser`
```

Comment:
```suggestion
* :doc:`/book/customers/customer_and_shop_user`
```
? 😄

Reply:
the file is named like that already, can be changed but ina different PR.